packages/host/app/components/ai-assistant/panel.gts

```diff
@@ -84,23 +84,23 @@ export default class AiAssistantPanel extends Component<Signature> {
       <div
         class='ai-assistant-panel'
         data-test-ai-assistant-panel
+        data-test-room-has-messages={{if this.currentRoom.messages true false}}
+        data-test-room-is-empty={{if this.currentRoom.messages false true}}
         ...attributes
       >
         <@resizeHandle />
         <header class='panel-header'>
-          {{#if this.currentRoom.messages}}
-            <div class='panel-title-group'>
-              <img
-                alt='AI Assistant'
-                src={{assistantIcon}}
-                width='20'
-                height='20'
-              />
-              <h3 class='panel-title-text' data-test-chat-title>
-                {{if this.currentRoom.name this.currentRoom.name 'Assistant'}}
-              </h3>
-            </div>
-          {{/if}}
+          <div class='panel-title-group'>
+            <img
+              alt='AI Assistant'
+              src={{assistantIcon}}
+              width='20'
+              height='20'
+            />
+            <h3 class='panel-title-text' data-test-chat-title>
```

Comment:
Some test assertions relied on `[data-test-chat-title]` being absent when the room had no messages. Since this is now stable while loading, the assertions now directly check whether a room is empty using the above test selectors.